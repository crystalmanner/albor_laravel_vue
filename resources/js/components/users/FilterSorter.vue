<template>
  <f-filter-sorter
    ref="filterSorter"
    placeholder="Search by first name, last name and email address"
    :expanded="expanded"
    @collapsed="expanded = false"
    @expanded="expanded = true"
    @run="run"
    @clear="clear"
  >
    <template v-slot:expanded="slotProps">
      <v-layout
        row
        wrap
      >
        <v-flex lg3>
          <v-layout row>
            <v-flex shrink>
              <v-card-text>Status</v-card-text>
            </v-flex>
            <v-flex>
              <v-select
                v-model="status"
                :items="statuses"
                item-text="name"
                item-value="id"
                placeholder="All"
                solo
                flat
                hide-details
                @change="slotProps.run"
              />
            </v-flex>
          </v-layout>
        </v-flex>

        <v-flex lg3>
          <v-layout row>
            <v-flex shrink>
              <v-card-text>Community</v-card-text>
            </v-flex>
            <v-flex>
              <f-multi-simple
                ref="communities"
                url="communities"
                term-param="filter[name]"
                placeholder="All"
                background-color="white"
                class="mt-0 pt-0"
                height="48"
                :value="community"
                @input="selectCommunities"
              />
            </v-flex>
          </v-layout>
        </v-flex>

        <v-flex lg3>
          <v-layout row>
            <v-flex shrink>
              <v-card-text>Discipline</v-card-text>
            </v-flex>
            <v-flex>
              <v-select
                v-model="discipline"
                :items="disciplines"
                placeholder="All"
                item-text="name"
                item-value="id"
                solo
                flat
                hide-details
                @change="slotProps.run"
              />
            </v-flex>
          </v-layout>
        </v-flex>

        <v-flex lg3>
          <v-layout row>
            <v-flex shrink>
              <v-card-text>Role</v-card-text>
            </v-flex>
            <v-flex>
              <f-multi-simple
                ref="roles"
                url="roles"
                term-param="filter[name]"
                placeholder="All"
                background-color="white"
                class="mt-0 pt-0"
                height="48"
                :value="role"
                @input="selectRoles"
              />
            </v-flex>
          </v-layout>
        </v-flex>
      </v-layout>
    </template>
  </f-filter-sorter>
</template>

<script>
import FFilterSorter from '@freshinup/core-ui/src/components/FFilterSorter.vue'
import FMultiSimple from '@freshinup/core-ui/src/components/FMultiSimple.vue'

export default {
  components: {
    FFilterSorter,
    FMultiSimple
  },
  data () {
    return {
      expanded: false,

      disciplines: [
        { id: 1, name: 'Accounting' },
        { id: 2, name: 'Communication' },
        { id: 3, name: 'Dining' },
        { id: 4, name: 'Engagement' },
        { id: 5, name: 'Maintenance' },
        { id: 6, name: 'Operations' },
        { id: 7, name: 'People' },
        { id: 8, name: 'Resident Care' },
        { id: 9, name: 'Sales' }
      ],
      discipline: null,

      statuses: [
        { id: 1, name: 'Active' },
        { id: 4, name: 'Pending' },
        { id: 3, name: 'On hold' }
      ],
      status: null,

      community: [],
      role: []
    }
  },
  methods: {
    getRunParams (params) {
      return {
        term: params.term,
        sort: params.orderBy,
        'filter[discipline_id]': this.discipline,
        'filter[status]': this.status,
        'filter[community_id]': this.community.map(item => item.id).join(','),
        'filter[arbor_role_id]': this.role.map(item => item.id).join(',')
      }
    },
    run (value) {
      this.$emit('runFilter', this.getRunParams(value))
    },
    clear (value) {
      this.discipline = null
      this.status = null
      this.community = []
      this.role = []
      this.$refs.roles.resetTerm()
      this.$refs.communities.resetTerm()

      this.run(value)
    },
    selectCommunities (communities) {
      this.community = communities || []
      this.$refs.filterSorter.run()
    },
    selectRoles (roles) {
      this.role = roles || []
      this.$refs.filterSorter.run()
    }
  }
}
</script>

<style lang="css" scoped>
/deep/ .v-text-field {
  padding-top: 0 !important;
  margin-top: 0 !important;
}
</style>
