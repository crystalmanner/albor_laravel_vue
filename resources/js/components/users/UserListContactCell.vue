<template>
  <div>
    {{ item.first_name }} {{ item.last_name }}
    <br>
    <small>{{ item.email }}</small>
  </div>
</template>

<script>
export default {
  props: {
    item: {
      type: Object,
      default: () => ({})
    }
  }
}
</script>
