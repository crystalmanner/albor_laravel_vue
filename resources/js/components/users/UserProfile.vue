<script>
import Component from 'fresh-bus/components/users/UserProfile.vue'

export default {
  components: {
    CompanyInfo: {
      render: () => '<div/>'
    },
    TeamsList: {
      render: () => '<div/>'
    }
  },
  extends: Component
}
</script>
