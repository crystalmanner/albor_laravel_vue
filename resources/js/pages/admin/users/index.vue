<script>
import Page from 'fresh-bus/pages/admin/users/index.vue'
import userFilter from '~/components/users/FilterSorter.vue'

export default {
  components: {
    userFilter
  },
  extends: Page,
  layout: Page.layout,
  beforeRouteEnterOrUpdate: Page.beforeRouteEnterOrUpdate
}
</script>
