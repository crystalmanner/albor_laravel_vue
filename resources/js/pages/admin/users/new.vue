<script>
import Page from 'fresh-bus/pages/admin/users/new.vue'

export default {
  extends: Page,
  layout: Page.layout,
  beforeRouteEnterOrUpdate: Page.beforeRouteEnterOrUpdate
}
</script>
