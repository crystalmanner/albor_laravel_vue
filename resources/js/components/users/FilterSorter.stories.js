import { storiesOf } from '@storybook/vue'
import { action } from '@storybook/addon-actions'
import mock, { proxy } from 'xhr-mock'

import FilterSorter from './FilterSorter'

const FIXTURE_ROLES = [
  { id: 1, name: 'Accountants' },
  { id: 2, name: 'Business Office Director' },
  { id: 3, name: 'Business Office Specialist' },
  { id: 4, name: 'Dining Director' },
  { id: 5, name: 'Dining Partner' },
  { id: 6, name: 'Dining Specialist' },
  { id: 7, name: 'Discipline Leader Accounting' },
  { id: 8, name: 'Discipline Leader Communication' },
  { id: 9, name: 'Discipline Leader Dining' },
  { id: 10, name: 'Discipline Leader Engagement' },
  { id: 12, name: 'Discipline Leader Resident Care' },
  { id: 13, name: 'Discipline leader Sales' },
  { id: 14, name: 'Engagement Director' },
  { id: 15, name: 'Engagement Partner' },
  { id: 16, name: 'Engagement Specialist' },
  { id: 17, name: 'EVP/People' },
  { id: 18, name: 'Executive Director' },
  { id: 19, name: 'Executive Project Coordinator' },
  { id: 20, name: 'HR Specialist' },
  { id: 21, name: 'Maintenance Specialist' },
  { id: 22, name: 'Marketing Specialist' },
  { id: 23, name: 'Memory Care Director' },
  { id: 24, name: 'Miantenance Director ' },
  { id: 25, name: 'Operations Partner' },
  { id: 26, name: 'President/Operations' },
  { id: 27, name: 'Resident Care Director' },
  { id: 28, name: 'Resident Care Partner' },
  { id: 29, name: 'Resident Care Specialist' },
  { id: 30, name: 'Sales Partners' },
  { id: 31, name: 'Sales Specialists' },
  { id: 32, name: 'Senior Care Counselor' },
  { id: 33, name: 'Training Specialist' }
]

const FIXTURE_COMMUNITIES = [
  { id: 1, name: 'Arbor Terrace at Cascade' },
  { id: 2, name: 'Arbor Terrace at Citrus Park' },
  { id: 3, name: 'Arbor Terrace at Crabapple' },
  { id: 4, name: 'Arbor Terrace at Hamilton Mill' },
  { id: 5, name: 'Arbor Terrace at Hamilton Mill IL' },
  { id: 6, name: 'Arbor Terrace at Kingwood Town Center' },
  { id: 7, name: 'Arbor Terrace Fairfax' },
  { id: 8, name: 'Arbor Terrace Fulton' },
  { id: 9, name: 'Arbor Terrace Lakeway' },
  { id: 10, name: 'Arbor Terrace Morris Plains' },
  { id: 11, name: 'Arbor Terrace Mountainside' },
  { id: 12, name: 'Arbor Terrace Mt Laurel' },
  { id: 13, name: 'Arbor Terrace Naperville' },
  { id: 14, name: 'Arbor Terrace of Asheville' },
  { id: 15, name: 'Arbor Terrace of Athens' },
  { id: 16, name: 'Arbor Terrace of Burnt Hickory' },
  { id: 17, name: 'Arbor Terrace of Decatur' },
  { id: 18, name: 'Arbor Terrace of East Cobb' },
  { id: 19, name: 'Arbor Terrace of Herndon' },
  { id: 20, name: 'Arbor Terrace of Johns Creek' },
  { id: 21, name: 'Arbor Terrace of Knoxville' },
  { id: 22, name: 'Arbor Terrace of Middletown' },
  { id: 23, name: 'Arbor Terrace Ortega' },
  { id: 24, name: 'Arbor Terrace Peachtree City' },
  { id: 25, name: 'Arbor Terrace Ponte Vedra' },
  { id: 26, name: 'Arbor Terrace Roseland' },
  { id: 27, name: 'Arbor Terrace San Jose' },
  { id: 28, name: 'Arbor Terrace Senior Living' },
  { id: 29, name: 'Arbor Terrace Shrewsbury' },
  { id: 30, name: 'Arbor Terrace Sudley Manor' },
  { id: 31, name: 'Arbor Terrace Teaneck' },
  { id: 32, name: 'Arbor Terrace Waugh Chapel' },
  { id: 33, name: 'Arbor Terrace Willistown' },
  { id: 34, name: 'Barrington Terrace of Fort Myers' },
  { id: 35, name: 'Barrington Terrace of Naples' },
  { id: 36, name: 'Eden Terrace of Spartanburg' },
  { id: 37, name: 'Renaissance on Peachtree' },
  { id: 38, name: 'Solana East Cobb' },
  { id: 39, name: 'Summit of Uptown' },
  { id: 40, name: 'The Arbor at BridgeMill' },
  { id: 41, name: 'The Arbor Company' },
  { id: 42, name: 'The Gardens at Eastside' },
  { id: 43, name: 'The Lakeside at Amelia ' },
  { id: 44, name: 'The Preserve at Palm Aire' },
  { id: 45, name: 'The Vantage at City View' }
]

mock.setup()
mock.get(/roles.*/, (req, res) => {
  return res.status(200).body(
    JSON.stringify({ data: FIXTURE_ROLES })
  )
})
mock.get(/communities.*/, (req, res) => {
  return res.status(200).body(
    JSON.stringify({ data: FIXTURE_COMMUNITIES })
  )
})
mock.use(proxy)

storiesOf(`Users|FilterSorter`, module)
  .add('mocked', () => ({
    components: { FilterSorter },
    template: `
      <v-container>
        <filter-sorter
          @runFilter="onRun"
        />
      </v-container>
    `,
    methods: {
      onRun (values) {
        action('run')(values)
      }
    }
  }))
