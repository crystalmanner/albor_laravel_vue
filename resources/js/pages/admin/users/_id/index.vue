<script>
import Page from 'fresh-bus/pages/admin/users/_id/index.vue'
import UserProfile from '~/components/users/UserProfile'

export default {
  components: {
    UserProfile
  },
  extends: Page,
  layout: Page.layout,
  beforeRouteEnterOrUpdate: Page.beforeRouteEnterOrUpdate
}
</script>
