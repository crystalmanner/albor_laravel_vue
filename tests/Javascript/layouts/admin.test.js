import { mount } from '@vue/test-utils'
import { createStore } from 'fresh-bus/store'
import { createLocalVue } from 'fresh-bus/tests/utils'
import { FIXTURE_USER } from 'fresh-bus/tests/__data__/users'
import Component from '~/layouts/admin.vue'

describe('Admin layout', () => {
  let localVue, mock, store
  describe('mount', () => {
    beforeEach(async () => {
      store = createStore({
        currentUser: FIXTURE_USER,
        page: {
          title: 'Admin template'
        }
      })
      const vue = createLocalVue({ validation: true })
      localVue = vue.localVue
      mock = vue.mock
        .onGet('api/currentUser').reply(200, FIXTURE_USER)
        .onGet('api/users/1').reply(200, { data: FIXTURE_USER })
        .onAny().reply(config => {
          console.error('No mock match for ' + config.url, config)
          return [404, {}]
        })
    })
    afterEach(() => {
      mock.restore()
    })
    it('snapshot of layout', async () => {
      const wrapper = mount(Component, {
        localVue,
        store
      })
      await wrapper.vm.$store.dispatch('users/getItem', { params: { id: 1 } })
      await wrapper.vm.$store.dispatch('page/setLoading', false)
      await wrapper.vm.$nextTick()
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
