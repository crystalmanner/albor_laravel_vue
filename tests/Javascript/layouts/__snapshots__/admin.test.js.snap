// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin layout mount snapshot of layout 1`] = `
<div
  class="page-layout"
>
  <aside
    class="v-navigation-drawer v-navigation-drawer--fixed v-navigation-drawer--open theme--dark"
    style="height: 100%; margin-top: 0px; max-height: calc(100% - 32px); transform: translateX(0px); width: 220px;"
  >
    <div
      class="logo-container"
    >
      <img
        src="https://freshinup.com/wp-content/uploads/2018/08/logo_Freshinup_final-1.png"
      />
    </div>
     
    <div
      class="v-list theme--dark"
      role="list"
    >
      <div
        class="v-list__group"
      >
        <div
          class="v-list__group__header"
        >
          <div
            class="v-list__group__header__prepend-icon"
          >
            <i
              aria-hidden="true"
              class="v-icon material-icons theme--dark"
            >
              dashboard
            </i>
          </div>
          <div
            role="listitem"
          >
            <div
              class="v-list__tile theme--dark"
            >
              <div
                class="v-list__tile__content"
              >
                <div
                  class="v-list__tile__title"
                >
                  Dashboard
                </div>
              </div>
            </div>
          </div>
          <div
            class="v-list__group__header__append-icon"
          >
            <i
              aria-hidden="true"
              class="v-icon material-icons theme--dark"
            >
              keyboard_arrow_down
            </i>
          </div>
        </div>
        <div
          class="v-list__group__items v-list__group__items--no-action"
          style="display: none;"
        >
           
          <div
            role="listitem"
          >
            <router-link
              class="v-list__tile v-list__tile--link theme--dark"
            >
              <div
                class="v-list__tile__content"
              >
                <div
                  class="v-list__tile__title"
                >
                  Home
                </div>
              </div>
               
              <div
                class="v-list__tile__action"
              >
                <i
                  aria-hidden="true"
                  class="v-icon material-icons theme--dark"
                />
              </div>
            </router-link>
          </div>
        </div>
      </div>
      <div
        class="v-list__group"
      >
        <div
          class="v-list__group__header"
        >
          <div
            class="v-list__group__header__prepend-icon"
          >
            <i
              aria-hidden="true"
              class="v-icon material-icons theme--dark"
            >
              supervised_user_circle
            </i>
          </div>
          <div
            role="listitem"
          >
            <div
              class="v-list__tile theme--dark"
            >
              <div
                class="v-list__tile__content"
              >
                <div
                  class="v-list__tile__title"
                >
                  Base User
                </div>
              </div>
            </div>
          </div>
          <div
            class="v-list__group__header__append-icon"
          >
            <i
              aria-hidden="true"
              class="v-icon material-icons theme--dark"
            >
              keyboard_arrow_down
            </i>
          </div>
        </div>
        <div
          class="v-list__group__items v-list__group__items--no-action"
          style="display: none;"
        >
           
          <div
            role="listitem"
          >
            <router-link
              class="v-list__tile v-list__tile--link theme--dark"
            >
              <div
                class="v-list__tile__content"
              >
                <div
                  class="v-list__tile__title"
                >
                  Users
                </div>
              </div>
               
              <div
                class="v-list__tile__action"
              >
                <i
                  aria-hidden="true"
                  class="v-icon material-icons theme--dark"
                />
              </div>
            </router-link>
          </div>
          <div
            role="listitem"
          >
            <router-link
              class="v-list__tile v-list__tile--link theme--dark"
            >
              <div
                class="v-list__tile__content"
              >
                <div
                  class="v-list__tile__title"
                >
                  Teams
                </div>
              </div>
               
              <div
                class="v-list__tile__action"
              >
                <i
                  aria-hidden="true"
                  class="v-icon material-icons theme--dark"
                />
              </div>
            </router-link>
          </div>
          <div
            role="listitem"
          >
            <router-link
              class="v-list__tile v-list__tile--link theme--dark"
            >
              <div
                class="v-list__tile__content"
              >
                <div
                  class="v-list__tile__title"
                >
                  Companies
                </div>
              </div>
               
              <div
                class="v-list__tile__action"
              >
                <i
                  aria-hidden="true"
                  class="v-icon material-icons theme--dark"
                />
              </div>
            </router-link>
          </div>
        </div>
      </div>
      <div
        class="v-list__group"
      >
        <div
          class="v-list__group__header"
        >
          <div
            class="v-list__group__header__prepend-icon"
          >
            <i
              aria-hidden="true"
              class="v-icon material-icons theme--dark"
            >
              settings
            </i>
          </div>
          <div
            role="listitem"
          >
            <div
              class="v-list__tile theme--dark"
            >
              <div
                class="v-list__tile__content"
              >
                <div
                  class="v-list__tile__title"
                >
                  Platform Settings
                </div>
              </div>
            </div>
          </div>
          <div
            class="v-list__group__header__append-icon"
          >
            <i
              aria-hidden="true"
              class="v-icon material-icons theme--dark"
            >
              keyboard_arrow_down
            </i>
          </div>
        </div>
        <div
          class="v-list__group__items v-list__group__items--no-action"
          style="display: none;"
        >
           
          <div
            role="listitem"
          >
            <router-link
              class="v-list__tile v-list__tile--link theme--dark"
            >
              <div
                class="v-list__tile__content"
              >
                <div
                  class="v-list__tile__title"
                >
                  Terms
                </div>
              </div>
               
              <div
                class="v-list__tile__action"
              >
                <i
                  aria-hidden="true"
                  class="v-icon material-icons theme--dark"
                />
              </div>
            </router-link>
          </div>
        </div>
      </div>
    </div>
    <div
      class="v-navigation-drawer__border"
    />
  </aside>
   
  <nav
    class="v-toolbar elevation-0 v-toolbar--fixed theme--light white"
    style="margin-top: 0px; padding-right: 0px; padding-left: 220px; transform: translateY(0px);"
  >
    <div
      class="v-toolbar__content"
      style="height: 64px;"
    >
      <div
        class="v-toolbar__title"
      >
        <div
          class="flex primary--text"
        >
          
            Arbor
          
        </div>
      </div>
       
      <div
        class="spacer"
      />
       
      <div
        class="v-menu v-menu--inline"
      >
        <div
          class="v-menu__activator v-menu__activator--disabled"
        >
          <button
            class="v-btn v-btn--disabled v-btn--flat v-btn--icon theme--light"
            disabled="disabled"
            type="button"
          >
            <div
              class="v-btn__content"
            >
              <span
                class="v-badge v-badge--overlap"
              >
                <!---->
                 
                <i
                  aria-hidden="true"
                  class="v-icon material-icons theme--light grey--text text--darken-2"
                  style="font-size: 28px;"
                >
                  notifications
                </i>
                <!---->
              </span>
            </div>
          </button>
        </div>
        <div
          class="v-menu__content theme--light "
          style="max-height: auto; min-width: 0px; max-width: 320px; top: 12px; left: 12px; transform-origin: right top; z-index: 0; display: none;"
        >
           
          <div
            class="v-card v-sheet theme--light"
          >
            <div
              class="v-list theme--light"
              role="list"
            />
          </div>
        </div>
      </div>
       
      <div
        class="v-menu v-menu--inline"
      >
        <div
          class="v-menu__activator"
        >
          <button
            class="avatar v-btn v-btn--flat v-btn--icon theme--light"
            type="button"
          >
            <div
              class="v-btn__content"
            >
              <div
                class="v-avatar"
                style="height: 48px; width: 48px;"
              >
                <img />
              </div>
            </div>
          </button>
        </div>
        <div
          class="v-menu__content theme--light user-menu"
          style="max-height: auto; min-width: 0px; max-width: 300px; top: 12px; left: 12px; transform-origin: top left; z-index: 0; display: none;"
        >
           
          <div
            class="v-card v-sheet theme--light"
          >
            <div
              class="layout pa-3 row"
            >
              <div
                class="flex shrink"
              >
                <div
                  class="v-avatar"
                  style="height: 64px; width: 64px;"
                >
                  <img />
                </div>
              </div>
               
              <div
                class="flex pl-3"
              >
                <h4
                  class="title secondary--text"
                >
                  
        
      
                </h4>
                 
                <span
                  class="v-chip mx-0 pl-3 pr-3 f-userMenu__userLevel v-chip--disabled v-chip--small theme--light secondary white--text"
                  tabindex="-1"
                >
                  <span
                    class="v-chip__content"
                  >
                    
        
      
                  </span>
                </span>
                 
                <div>
                  <!---->
                  
          
        
                </div>
              </div>
            </div>
             
            <hr
              class="v-divider theme--light"
            />
             
            <div
              class="v-list pb-0 theme--light"
              role="list"
            >
              <div
                class="secondary--text darken-3"
                role="listitem"
              >
                <router-link
                  class="v-list__tile v-list__tile--link theme--light"
                >
                  <div
                    class="v-list__tile__content"
                  >
                    <div
                      class="v-list__tile__title"
                    >
                      My Profile
                    </div>
                  </div>
                </router-link>
              </div>
              <div
                class="secondary--text darken-3"
                role="listitem"
              >
                <router-link
                  class="v-list__tile v-list__tile--link theme--light"
                >
                  <div
                    class="v-list__tile__content"
                  >
                    <div
                      class="v-list__tile__title"
                    >
                      My Teams
                    </div>
                  </div>
                </router-link>
              </div>
              <div
                class="secondary--text darken-3"
                role="listitem"
              >
                <router-link
                  class="v-list__tile v-list__tile--link theme--light"
                >
                  <div
                    class="v-list__tile__content"
                  >
                    <div
                      class="v-list__tile__title"
                    >
                      My Company
                    </div>
                  </div>
                </router-link>
              </div>
              <div
                class="secondary--text darken-3"
                role="listitem"
              >
                <router-link
                  class="v-list__tile v-list__tile--link theme--light"
                >
                  <div
                    class="v-list__tile__content"
                  >
                    <div
                      class="v-list__tile__title"
                    >
                      My Settings
                    </div>
                  </div>
                </router-link>
              </div>
              <div
                class="secondary--text darken-3"
                role="listitem"
              >
                <router-link
                  class="v-list__tile v-list__tile--link theme--light"
                >
                  <div
                    class="v-list__tile__content"
                  >
                    <div
                      class="v-list__tile__title"
                    >
                      My Company Settings
                    </div>
                  </div>
                </router-link>
              </div>
               
              <!---->
               
              <hr
                class="v-divider theme--light"
              />
               
              <div
                role="listitem"
              >
                <a
                  class="v-list__tile v-list__tile--link theme--light"
                >
                  <div
                    class="v-list__tile__title"
                  >
                    Sign Out
                  </div>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </nav>
   
  <main
    class="v-content main-page-container"
    style="padding: 64px 0px 32px 220px;"
  >
    <div
      class="v-content__wrap"
    >
      <!---->
       
      <router-view
        style=""
      />
       
      <!---->
       
      <!---->
    </div>
  </main>
   
  <footer
    class="v-footer white--text v-footer--fixed theme--light blue-grey"
    style="height: 32px;"
  >
    <span>
      Powered by FreshinUp
    </span>
     
    <div
      class="spacer"
    />
     
    <span>
      © 2019
    </span>
  </footer>
</div>
`;
